actioncable/README.md

```diff
@@ -526,6 +526,15 @@ cable.subscriptions.create 'AppearanceChannel',
     # normal channel code goes here...
 ```
 
+## Download and Installation
+
+The latest version of Action Cable can be installed with [RubyGems](#gem-usage),
```

Comment:
@maclover7 I mean similar to how npm has npm install at https://github.com/maclover7/rails/blob/eff98450ff17855439419db25976d2ecfe54dc04/actioncable/README.md#installation , there should be a "Installation" section in https://github.com/maclover7/rails/blob/eff98450ff17855439419db25976d2ecfe54dc04/actioncable/README.md#gem-usage

Right now, the RG section does not have this information.